### PR TITLE
Make coordinates of vector data chunks independent from scene origin

### PR DESCRIPTION
**Pull request by wonder-sk**
This is a continuation of work started in #58948, handling all the cases of data from vector layers. No change of functionality, just changing what coordinates are stored in geometries sent to GPU.

Also:
- point and line symbol implementations internally use (x,y,z) coordinates instead of (x,z,-y) - QTransforms are used to flip the axes back to scene's coordinates
- instanced renderer's vertex shader has been cleaned up and made to work with QTransform
- extruded polygons textured rendering test was modified to use more appropriately scaled texture
- unused QgsTessellatedPolygonGeometry::setPolygons() was removed

### CODE REVIEW

**Comment by wonder-sk** [comment]
This should be ready for review...

File: python/3d/auto_generated/symbols/qgspoint3dsymbol.sip.in [hunk 1]
```diff
@@ -191,9 +191,9 @@ Returns transform for individual objects represented by the symbol
 Sets transform for individual objects represented by the symbol
 %End
 
-    QMatrix4x4 billboardTransform() const;
+    float billboardHeight() const;
 %Docstring
-Returns transform for billboards
+Returns how nuch the billboard should be elevated upwards
```
Comment (nyalldawson) [inline python/3d/auto_generated/symbols/qgspoint3dsymbol.sip.in:196]:
```suggestion
Returns how much the billboard should be elevated upwards
```

**Review by nyalldawson — APPROVED** [review]
Looks great!

File: python/PyQt6/3d/auto_generated/symbols/qgspoint3dsymbol.sip.in [hunk 2]
```diff
@@ -191,9 +191,9 @@ Returns transform for individual objects represented by the symbol
 Sets transform for individual objects represented by the symbol
 %End
 
-    QMatrix4x4 billboardTransform() const;
+    float billboardHeight() const;
 %Docstring
-Returns transform for billboards
+Returns how nuch the billboard should be elevated upwards
```
Comment (wonder-sk) [inline python/PyQt6/3d/auto_generated/symbols/qgspoint3dsymbol.sip.in:196]:
```suggestion
Returns how much the billboard should be elevated upwards
```

File: src/3d/symbols/qgspoint3dsymbol.h [hunk 3]
```diff
@@ -164,8 +164,8 @@ class _3D_EXPORT QgsPoint3DSymbol : public QgsAbstract3DSymbol SIP_NODEFAULTCTOR
     //! Sets transform for individual objects represented by the symbol
     void setTransform( const QMatrix4x4 &transform ) { mTransform = transform; }
 
-    //! Returns transform for billboards
-    QMatrix4x4 billboardTransform() const;
+    //! Returns how nuch the billboard should be elevated upwards
```
Comment (wonder-sk) [inline src/3d/symbols/qgspoint3dsymbol.h:167]:
```suggestion
    //! Returns how much the billboard should be elevated upwards
```

**Pull request merged**